src/api.ts

```diff
@@ -0,0 +1,17 @@
+import { Hono } from "hono";
+
+export const api = new Hono();
+
+api.get("/board", async (c) => {
+	const redis = c.get("redis");
+	const board = await redis.get("board");
+	return c.json({ board });
+});
+
+api.get("/board/:offset", async (c) => {
+	const offset = c.req.param("offset");
+	const redis = c.get("redis");
+	const board = await redis.bitfield("board", "GET", "u8", offset);
+
+	return c.json({ board });
```

Comment:
this isn't technically `board` here, `value` would be more accurate.
also, in my testing, `board` is `[number]`. even when `offset` is invalid, it returns `{ board: [0] }`. no way to differentiate since `0` is a valid value (maybe)

Reply:
yeah we can add zod validation for offset. Like checking if it's multiples of 8 (0,8,16,32...).

Reply:
eventually